From: Michal Suchanek <msuchanek@suse.de>
Subject: Add back sibling paca poiter to paca

References: bsc#1055117
Patch-mainline: no, kabi

Commit 2bf1071a8d50 ("powerpc/64s: Remove POWER9 DD1 support") removes
thread_sibling_pacas member from paca. To preserve kABI add back the member.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>

--- a/arch/powerpc/include/asm/paca.h
+++ b/arch/powerpc/include/asm/paca.h
@@ -198,6 +198,7 @@ struct paca_struct {
 	u8 subcore_sibling_mask;
 	/* Flag to request this thread not to stop */
 	atomic_t dont_stop;
+	struct paca_struct **thread_sibling_pacas;
 	/* The PSSCR value that the kernel requested before going to stop */
 	u64 requested_psscr;
 
